ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/InsertValuesExecutor.java

```diff
@@ -224,6 +226,14 @@ private GenericRow extractRow(
     private final Schema schema;
     private final String field;
 
+    private static final Map<Type, Function<Number, Object>> CASTER =
```

Comment:
I think this would fit better in SchemaUtil. Its a bit weird to ask SchemaUtil if the type can be up-casted, but actually do the casting here. Maybe just have a single function in SchemaUtil called `maybeUpCast`?